bucket/flow-launcher.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "1.9.1",
+    "description": "Quick file search & app launcher for Windows with community-made plugins",
```

Comment:
Let's just keep the description as it is.

Reply:
"Quick file search" is a verb. "App launcher" is a noun. If you want to use "and" to join them, either change both of them to verbs or bot

Reply:
Why? The changes are requested for a reason. Think about it.

Reply:
Maybe we should leave it? It’s the same as the about box on their GitHub page

Reply:
It's what we are using at the moment and reads fine, doesn't have to be 100% grammatically accurate.

Reply:
> doesn't have to be 100% grammatically accurate.

Well, that's definitely a first.

You can keep it like that, but replace '&' with 'and' and remove the 'for Windows' part.

Reply:
Does it really matter that much?

Reply:
Yes. Scoop is not available for any other OS. The "for Windows" part is redundant. Why are you so hesitant to change it? The descriptions should concise and as short and sufficient as possible (they should also be grammatically correct, but...).

Consistency between all the manifest in an official bucket is key. If there's a rogue '&' symbol (it's not really saving space here, is it?) or a grammatical mistake, it sheds a not-so-good light on both the application developers and the bucket maintainers.

Reply:
> The "for Windows" part is redundant

https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/hain.json
https://github.com/ScoopInstaller/Extras/blob/78d54c34470f95332d7fb6200d31e0f9966f85eb/bucket/staxrip.json
https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/advancedrenamer.json
https://github.com/ScoopInstaller/Extras/blob/0f723418a2d28bdbb27f834991d936a1dc3fb9ea/bucket/hourglass.json
https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/mpv.net.json
https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/tortoisemerge.json
https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/pasteboard.json
https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/bitsmanager.json

> If there's a rogue '&' symbol (it's not really saving space here, is it?) or a grammatical mistake

https://github.com/ScoopInstaller/Extras/blob/f5f097f5427533a2407dadd787fb731af8068198/bucket/brave.json
https://github.com/ScoopInstaller/Extras/blob/e30a943e7726ee8dcbeffd6f9a965065c6b5f30d/bucket/languagetool.json
https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/aria-ng-gui.json
https://github.com/ScoopInstaller/Extras/blob/edf69cf51bdb79b1faca4bf3491ef31eb20eac27/bucket/adwcleaner.json
https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/android-sms-backup-and-restore.json

> Why are you so hesitant to change it?

Because the description is currently used on our homepage and website, I think you are nitpicking and is not necessary.